New version: GIGABYTE.GigabyteControlCenter version 25.02.07.01

Dragon1573 opened the pull request:
## Checklist for Pull Requests

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Resolve #230240
  - [x] Depend on #230241

## Manifests

- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.9 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.9.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

## Manual validation

![Image](https://github.com/user-attachments/assets/0b68d463-c420-49b4-a791-71922b63f73f)

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/230284)

Dragon1573 committed:
New version: GIGABYTE.GigabyteControlCenter version 25.02.07.01

Signed-off-by: Dragon1573 <49941141+Dragon1573@users.noreply.github.com>

DuckDuckStudio commented on manifests/g/GIGABYTE/GigabyteControlCenter/25.02.07.01/GIGABYTE.GigabyteControlCenter.installer.yaml:
```diff
@@ -0,0 +1,34 @@
+      - RelativeFilePath: GIGABYTE Control Center_2025_Jan_release_All_Setup_B25.02.07.01.exe
+    InstallerUrl: https://download.gigabyte.com/FileList/Utility/GCC_25.02.07.01.zip
+    InstallerSha256: F7F0C2E08DC2AD7953260D42F01086AB7FF52908618B84A146E3224235333FD5
+  - InstallerLocale: zh-CN
+    Architecture: x86
+    NestedInstallerType: exe
+    NestedInstallerFiles:
+      - RelativeFilePath: GIGABYTE Control Center_2025_Jan_release_All_Setup_B25.02.07.01.exe
+    InstallerUrl: https://download.gigabyte.cn/FileList/Utility/GCC_25.02.07.01.zip
+    InstallerSha256: F7F0C2E08DC2AD7953260D42F01086AB7FF52908618B84A146E3224235333FD5
```
```suggestion
```

在中国大陆，似乎使用`.com`的链接下载更快。
- Related to https://github.com/microsoft/winget-pkgs/pull/201791#issuecomment-2561379165

DuckDuckStudio commented on manifests/g/GIGABYTE/GigabyteControlCenter/25.02.07.01/GIGABYTE.GigabyteControlCenter.installer.yaml:
```diff
@@ -0,0 +1,34 @@
+AppsAndFeaturesEntries:
+  - DisplayName: GIGABYTE Control Center 25.01.20.01
+    Publisher: GIGABYTE
+    DisplayVersion: 25.02.07.01
```
```suggestion
    DisplayVersion: 25.01.20.01
```

您能否再检查一下此条目？我(在没有测试的情况下)认为是这个

Dragon1573 replied:
✅ 已确认，之前截图里显示的版本号也是这个。

😕 非常不理解为什么安装后显示的版本号和 `InstallerUrl` 中标注的版本号不一样。

- 参见 <https://github.com/microsoft/winget-pkgs/pull/230284#issue-2856223464>

DuckDuckStudio commented:
> Url Validation Error
> 
> * manifests\g\GIGABYTE\GigabyteControlCenter\25.02.07.01
>   
>   * https://www.gigabyte.cn
>     
>     * Http status code: Forbidden
>   * https://www.gigabyte.cn/Consumer/Software/GIGABYTE-Control-Center/
>     
>     * Http status code: Forbidden

It looks like it's a false alarm or a regional block.
![image](https://github.com/user-attachments/assets/3f28d8ed-e90f-4611-887b-d4e9664d1e85)
![image](https://github.com/user-attachments/assets/bc84741f-3463-4eca-8515-88323c004679)
When you visit `https://www.gigabyte.com` url in mainland China, you will be automatically redirected to `https://www.gigabyte.cn` url.

Dragon1573 committed:
Apply suggestions from code review

- Remove Mainland China specific installer entrance
  - The unified ".com" domain is not blocked in China
  - In Mainland China, ".com" will be automatically redirected to ".cn"
  - ".cn" connection speed is slightly slower than ".com"

Co-authored-by: 鸭鸭「カモ」 <89643991+DuckDuckStudio@users.noreply.github.com>

Dragon1573 committed:
Apply suggestions from code review

- Correction of "DisplayVersion" property to match that in screenshot

Co-authored-by: 鸭鸭「カモ」 <89643991+DuckDuckStudio@users.noreply.github.com>

Dragon1573 commented:
> Hello @Dragon1573,
> 
> The package manager bot determined there was an issue with installing the application correctly. Please check the application installs correctly. Once repaired, please push an update to your pull request.
> 
> Template: msftbot/validationError/installation/general

This problem should be suppressed.

- See <https://github.com/microsoft/winget-pkgs/pull/230284#issuecomment-2662077664>

stephengillie commented:
Automatic Validation ended with:
>  Installation failed with exit code -1978335229 2025-02-17 01:08:06.394 [CLI ] Caught std::exception: bad allocation

Automated error analysis suggests -1978335229 may mean Executing command failed

(Automated response - build 980.)

stephengillie commented:
> > Hello @Dragon1573,
> > The package manager bot determined there was an issue with installing the application correctly. Please check the application installs correctly. Once repaired, please push an update to your pull request.
> > Template: msftbot/validationError/installation/general
> 
> This problem should be suppressed.
> 
> * See [New version: GIGABYTE.GigabyteControlCenter version 25.02.07.01 #230284 (comment)](https://github.com/microsoft/winget-pkgs/pull/230284#issuecomment-2662077664)
![image](https://github.com/user-attachments/assets/553d5097-d87f-4514-a679-da0b302a14c0)
![image](https://github.com/user-attachments/assets/7108c5e8-c2b3-4d71-9d4c-17b96f68fadb)

stephengillie commented on manifests/g/GIGABYTE/GigabyteControlCenter/25.02.07.01/GIGABYTE.GigabyteControlCenter.locale.zh-CN.yaml:
```diff
@@ -0,0 +1,28 @@
+PackageVersion: 25.02.07.01
+PackageLocale: zh-CN
+Publisher: 技嘉科技股份有限公司
+PublisherUrl: https://www.gigabyte.cn
```
```suggestion
# PublisherUrl: 
```

DuckDuckStudio replied:
My suggested:
```suggestion
PublisherUrl: https://www.gigabyte.com
```

> When you visit https://www.gigabyte.com in mainland China, you will be automatically redirected to https://www.gigabyte.cn.

stephengillie commented on manifests/g/GIGABYTE/GigabyteControlCenter/25.02.07.01/GIGABYTE.GigabyteControlCenter.locale.zh-CN.yaml:
```diff
@@ -0,0 +1,28 @@
+PrivacyUrl: https://profile.gigabyte.com/PrivacyPolicy.aspx?l=zh-CN
+Author: 技嘉科技股份有限公司
+PackageName: Gigabyte Control Center (GCC)
+PackageUrl: https://www.gigabyte.cn/Consumer/Software/GIGABYTE-Control-Center/
```
```suggestion
#PackageUrl: 
```

DuckDuckStudio replied:
My suggestion:
```suggestion
PublisherUrl: https://www.gigabyte.com/Consumer/Software/GIGABYTE-Control-Center/
```

> When you visit https://www.gigabyte.com in mainland China, you will be automatically redirected to https://www.gigabyte.cn.

Dragon1573 committed:
Apply suggestions from code review

- Switch domain suffix of the package
- Prevent false positive report of region blocking

Co-authored-by: 鸭鸭「カモ」 <89643991+DuckDuckStudio@users.noreply.github.com>

DuckDuckStudio commented:
> Automatic Validation ended with:
> >  Installation failed with exit code -1978335229 2025-02-17 01:08:06.394 [CLI ] Caught std::exception: bad allocation
> 
> Automated error analysis suggests -1978335229 may mean Executing command failed
> 
> (Automated response - build 980.)

I don't think this package executes by command...

stephengillie commented:
Automatic Validation ended with:
>  Installation failed with exit code -1978335229 2025-02-17 22:54:37.115 [CLI ] Caught std::exception: bad allocation

Automated error analysis suggests -1978335229 may mean Executing command failed

(Automated response - build 981.)

DuckDuckStudio commented:
Reply https://github.com/microsoft/winget-pkgs/pull/230284#issuecomment-2676411934

ExtLIB_Loader_Lib seems to be java related.

DuckDuckStudio reviewed (commented):
Verify:
Installer: zip + 7z sfx
![image](https://github.com/user-attachments/assets/8ddc14d2-f9f2-44bb-8d06-6f2a1c9a0b7e)
![image](https://github.com/user-attachments/assets/8d79a456-4b40-4b14-8084-0ca1b4969cbd)
No errors or warning messages.

Dragon1573 commented:
I think this is expected.

There might be some special dependencies that prevent this package from running properly in a VM or Windows Sandbox instance.
